### PR TITLE
[Translation] Multiple Gitconfig article to ID

### PR DESCRIPTION
**Pull request by paramithatm**
## What
- Translated article: working-with-multiple-gitconfig.md
- Replace pronoun `Anda` with `kalian` in docker context article
- Fix wrong formatting in work life balance article

### CODE REVIEW

File: content/id/blogs/working-with-multiple-gitconfig.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: "Bekerja menggunakan beberapa Gitconfig"
```
Comment (clavinjune) [inline content/id/blogs/working-with-multiple-gitconfig.md:2]: how about `bekerja menggunakan lebih dari 1 Gitconfig`? @paramithatm

File: content/id/blogs/working-with-multiple-gitconfig.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	name = foo bar
+	email = foo@company.com
+	signingKey = ABCDEF012345
+[commit]
+	gpgsign = true
+[core]
+	editor = vim
+```
+
+Sekarang kedua akun Anda sudah diatur. Bagaimana cara berganti antara kedua gitconfig?
```
Comment (clavinjune) [inline content/id/blogs/working-with-multiple-gitconfig.md:77]: `Bagaimana cara berpindah dari satu Gitconfig ke Gitconfig yang lain?` wdyt?

**Review by clavinjune — CHANGES_REQUESTED** [review]
how about we use `Kalian` instead of `Anda` when it comes to words `You`?

File: content/id/blogs/working-with-multiple-gitconfig.md (hunk 1 above)
Comment (paramithatm) [reply content/id/blogs/working-with-multiple-gitconfig.md]: updated

Comment (clavinjune) [reply content/id/blogs/working-with-multiple-gitconfig.md]: [second point](https://ejaan.kemdikbud.go.id/eyd/penulisan-kata/angka-dan-bilangan/)

Comment (paramithatm) [reply content/id/blogs/working-with-multiple-gitconfig.md]: done

File: content/id/blogs/working-with-multiple-gitconfig.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+aliases: []
+description: "pakai banyak gitconfig, pisahkan persona git kalian"
+---
+#### Translated by: [@paramithatm](https://github.com/paramithatm)
+
+{{< unsplash user="@praveentcom" src="photo-1647166545674-ce28ce93bdca" q="50" >}}
+
+## Pengenalan
+
+Ini adalah tips singkat untuk menggunakan beberapa git config pada 1 mesin lokal. Jika kalian bekerja menggunakan Gitlab/Bitbucket/Github dengan beberapa email/username/gpgsign yang berbeda, atau jika kalian menggunakan akun personal dan akun kerja kalian di mesin yang sama.
```
Comment (clavinjune) [inline content/id/blogs/working-with-multiple-gitconfig.md:18]:
```suggestion
Ini adalah tips singkat untuk menggunakan beberapa git config pada mesin lokal yang sama. Jika kalian bekerja menggunakan Gitlab/Bitbucket/Github dengan beberapa email/username/gpgsign yang berbeda, atau jika kalian menggunakan akun personal dan akun kerja kalian di mesin yang sama.
```

File: content/id/blogs/working-with-multiple-gitconfig.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+```shell
+$ mkdir -p "$HOME/.gitconfig.d/" && cd "$_"
+$ pwd
+/Users/clavianus.juneardo/.gitconfig.d
+```
+
+## Mengatur Masing-masing Gitconfig
+
+Setelah membuat direktori, misalnya kalian mau mengatur akun `personal` dan `kerja`:
```
Comment (clavinjune) [inline content/id/blogs/working-with-multiple-gitconfig.md:32]: quite weird here 🤔

File: content/id/blogs/working-with-multiple-gitconfig.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+### Mengatur Gitconfig Personal
+
+```shell
+$ git config --file=personal user.name "foo"
+$ git config --file=personal user.email "foo@gmail.com"
+$ git config --file=personal core.editor "vim"
+...
+```
+
+Dengan menggunakan penanda `--file=personal`, gitconfig tersebut akan dikonfigurasi di dalam _file_ `personal`.
```
Comment (clavinjune) [inline content/id/blogs/working-with-multiple-gitconfig.md:43]: *flag* is technical term, I don't think we need to translate it


```suggestion
Dengan menggunakan flag `--file=personal`, gitconfig tersebut akan dikonfigurasi di dalam _file_ `personal`.
```

**Review by clavinjune — CHANGES_REQUESTED** [review]

File: content/id/blogs/working-with-multiple-gitconfig.md (hunk 3 above)
Comment (paramithatm) [reply content/id/blogs/working-with-multiple-gitconfig.md:18]: done

File: content/id/blogs/working-with-multiple-gitconfig.md (hunk 5 above)
Comment (paramithatm) [reply content/id/blogs/working-with-multiple-gitconfig.md:43]: done

File: content/id/blogs/working-with-multiple-gitconfig.md (hunk 4 above)
Comment (paramithatm) [reply content/id/blogs/working-with-multiple-gitconfig.md:32]: agree.
```Setelah membuat direktori, misalnya kalian mau membuat akun `personal` dan `kerja` kalian:```
how about this?

**Review by clavinjune — APPROVED** [review]

**Pull request merged**